util/automation.py

```diff
@@ -32,8 +32,12 @@
 
 DEEPSPEECH_CLONE_PATH  = os.path.abspath(os.environ.get('ds_clone_path', './ds_exec_clone/'))
 
-CACHE_DIR = os.path.join(BaseDirectory.xdg_cache_home, 'deepspeech_wer')
-DATA_DIR  = os.path.join(BaseDirectory.xdg_data_home, 'deepspeech_wer')
+# Cache and data should be used for small amounts of data
+CACHE_DIR     = os.path.join(BaseDirectory.xdg_cache_home, 'deepspeech_wer')
+DATA_DIR      = os.path.join(BaseDirectory.xdg_data_home, 'deepspeech_wer')
```

Comment:
DATA_DIR is being used elsewhere, `src_logs_dir = os.path.join(DATA_DIR, 'logs')`